src/frontend/context/Kompetanse/KompetanseSkjemaContext.tsx

```diff
@@ -0,0 +1,191 @@
+            initielFom: initelFom,
+            barnIdenter: useFelt<OptionType[]>({
+                verdi: valgteBarn,
+                valideringsfunksjon: erBarnGyldig,
+            }),
+            periode: useFelt<IYearMonthPeriode>({
+                verdi: nyYearMonthPeriode(kompetanse.fom, kompetanse.tom),
+                avhengigheter: { initelFom },
+                valideringsfunksjon: erEøsPeriodeGyldig,
+            }),
+            søkersAktivitet: useFelt<SøkersAktivitet | undefined>({
+                verdi: kompetanse.søkersAktivitet,
+                valideringsfunksjon: erSøkersAktivitetGyldig,
+            }),
+            annenForeldersAktivitet: annenForeldersAktivitet,
+            annenForeldersAktivitetsland: useFelt<string | undefined>({
+                verdi: kompetanse.annenForeldersAktivitetsland,
+                avhengigheter: { annenForeldersAktivitet },
+                valideringsfunksjon: erAnnenForeldersAktivitetslandGyldig,
+            }),
+            barnetsBostedsland: useFelt<string | undefined>({
+                verdi: kompetanse.barnetsBostedsland,
+                valideringsfunksjon: erBarnetsBostedslandGyldig,
+            }),
+            resultat: useFelt<KompetanseResultat | undefined>({
+                verdi: kompetanse.resultat,
+                valideringsfunksjon: erKompetanseResultatGyldig,
+            }),
+        },
+        skjemanavn: kompetanseFeilmeldingId(kompetanse),
+    });
+
+    const sendInnSkjema = () => {
+        if (kanSendeSkjema()) {
+            settSubmitRessurs(byggTomRessurs());
+            settVisfeilmeldinger(false);
+            onSubmit(
+                {
+                    method: 'PUT',
+                    data: {
+                        id: kompetanse.id,
+                        fom: skjema.felter.periode.verdi.fom || '', // undefined fom vil bli stoppet i valideringen
+                        tom: skjema.felter.periode.verdi.tom,
+                        barnIdenter: skjema.felter.barnIdenter.verdi.map(barn => barn.value),
+                        søkersAktivitet: skjema.felter.søkersAktivitet.verdi,
+                        annenForeldersAktivitet: skjema.felter.annenForeldersAktivitet.verdi,
+                        annenForeldersAktivitetsland:
+                            skjema.felter.annenForeldersAktivitetsland.verdi,
+                        barnetsBostedsland: skjema.felter.barnetsBostedsland.verdi,
+                        resultat: skjema.felter.resultat.verdi,
+                    },
+                    url: `/familie-ba-sak/api/kompetanse/${behandlingId}`,
+                },
+                (response: Ressurs<IBehandling>) => {
+                    if (response.status === RessursStatus.SUKSESS) {
+                        nullstillSkjema();
+                        settErKompetanseEkspandert(false);
+                        settÅpenBehandling(response);
+                    }
+                }
```

Comment:
Vurdere å trekke ut denne også? Likt for `Kompetanse`, `UtenlandskPeriodebeløp` og `Valutakurs`.